testserv.c

```diff
@@ -343,6 +376,20 @@ cttest_too_long_commandline()
     ckresp(fd, "BAD_FORMAT\r\n");
 }
 
+void
+cttest_too_long_socketname()
+{
+    #define NAME "/tmp/aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+    progname = __func__;
+    struct sockaddr_un addr;
+    const size_t maxlen = sizeof(addr.sun_path);
+    char real_name[maxlen];
+    snprintf(real_name, maxlen, NAME);
```

Comment:
There's no %-format specifier, so please don't use snprintf(); use strlcpy() instead.

Reply:
Good point, although I'll stick to `snprintf`.  Remember that this is just test code, so the fact that it works triumphs over its elegance.